content/en/blog/_posts/2023-05-01-kmsv2-beta.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: blog
+title: "Kubernetes 1.27: KMS V2 Moves to Beta"
+date: 2023-05-01
+slug: kms-v2-moves-to-beta
+---
+
+**Authors:** Anish Ramasekar, Mo Khan, and Rita Zhang (Microsoft)
+
+With Kubernetes 1.27, we (SIG Auth) are moving Key Management Service (KMS) v2 API to beta.
+
+## What is KMS?
+One of the first things to consider when securing a Kubernetes cluster is encrypting etcd data at rest.
+KMS provides an interface for a provider to utilize a key stored in an external key service to perform this encryption.
+
+KMS v1 has been a feature of Kubernetes since version 1.10,
+and is currently in beta as of version v1.12.
+KMS v2 was introduced as alpha in v1.25.
+
+{{% alert title="Note" color="primary" %}}
+The KMS v2 API and implementation changed in incompatible ways in-between the alpha release in v1.25 and the beta release in v1.27. The design of KMS v2 has changed since [the previous blog post](https://kubernetes.io/blog/2022/09/09/kms-v2-improvements/) was written and it is not compatible with the design in this blog post. Attempting to upgrade from old versions with the alpha feature enabled will result in data loss.
```

Comment:
please wrap long lines like this ... e.g.

```suggestion
The KMS v2 API and implementation changed in incompatible ways betweenthe alpha release
in v1.25 and the beta release in v1.27. The design of KMS v2 has changed since
[the previous blog post](https://kubernetes.io/blog/2022/09/09/kms-v2-improvements/)
was written and it is not compatible with the design in this blog post.
Attempting to upgrade from old versions with the alpha feature enabled will result in data loss.
```

Manual wrapping helps the reviewers to provide feedbacks and for the downstream localization teams to track changes in a line-by-line way (that is how git diff works anyway).

Reply:
@tengqm I used the [Rewrap](https://stkb.github.io/Rewrap/) vscode plugin to wrap long lines. LMK what you think!